CHANGELOG.md

```diff
@@ -25,6 +25,7 @@ Please avoid adding duplicate information across this changelog and JIRA/doc inp
 
 - Slim/Full Collector images have been deprecated and will be removed in a
   future release. The two image flavors are now functionally identical (neither contain any kernel drivers.)
+- The `/v1/summary/counts` API has been deprecated in 4.5 and will be removed in the future. Use `api/graphql?opname=summary_counts` instead.
```

Comment:
We do not officially support GraphQL APIs today, they are for internal use (i.e. UI) only so I suggest we remove the recommendation. If you want to go the extra mile, you can check how the graphql operation works and suggest the alternatives from separate services.

Btw, have you checked `api/graphql?opname=summary_counts` works? I was under the impression it's a concept in our UI and translates to something different on the backend.

Reply:
+1 on not advocating GrahQL usage; while we know right now that a couple of folks are using it, it's not something we officially support (and maintain with the same backwards strictness as the REST APIs).

Reply:
I've removed the suggestion 👍 

> have you checked api/graphql?opname=summary_counts works?

Yes I did. It does not work exactly like that though. This is the real call you'd need to do:

```
curl -s -k -X POST -H "Content-Type: application/json" -H "Authorization: Bearer ${ROX_API_TOKEN}" -d "{\"operationName\":\"summary_counts\",\"variables\":{},\"query\":\"query summary_counts {\n  clusterCount\n  nodeCount\n  violationCount\n  deploymentCount\n  imageCount\n  secretCount\n}\"}" https://localhost:8000/api/graphql
```